.kitchen.yml

```diff
@@ -0,0 +1,72 @@
+---
+driver:
+  name: docker
+  #socket: tcp://127.0.0.1:2376
+  hostname: git.ci.local
+  use_sudo: true
+
+provisioner:
+  name: salt_solo
+  salt_install: bootstrap
+  salt_bootstrap_url: https://bootstrap.saltstack.com
+  salt_version: latest
+  require_chef: false
+  #init_environment: |
+  #  # for reclass
+  #  mkdir -p $SALT_ROOT/reclass/classes
+  #  ln -fs /usr/share/salt-formulas/reclass/* $SALT_ROOT/reclass/
+  #  find /usr/share/salt-formulas/env/_formulas -name metadata -type d | xargs -I'{}' \
+  #    ln -fs {}/service $SALT_ROOT/reclass/classes/
+  vendor_repo:
+    - type: apt
+      url: http://apt.tcpcloud.eu/nightly
+      key_url: http://apt.tcpcloud.eu/public.gpg
+      components: main tcp-salt
+  dependencies:
+    - name: linux
+      repo: apt
+      package: salt-formula-linux
+    # Test of git external dependencies that are not directly required
+    - name: nginx
+      repo: git
+      source: https://github.com/tcpcloud/salt-formula-nginx.git
```

Comment:
I would wait when we'll know final location of salt-formulas before doing tests for all of them. I would use git repository for deploying dependencies (or have some env-variable to decide?)

Also this is interesting (just FYI): https://docs.travis-ci.com/user/deployment/launchpad/
And we'll probably do that to have package builds.